sleap/gui/commands.py

```diff
@@ -1601,6 +1605,28 @@ class GoPrevSuggestedFrame(GoNextSuggestedFrame):
     seek_direction = -1
 
 
+class GoNextInstanceChange(NavCommand):
+    @classmethod
+    def do_action(cls, context: CommandContext, params: dict):
+        video = context.state["video"]
+        cur_idx = context.state["frame_idx"]
+
+        # Get current number of instances
+        current_frame = context.labels.find(video, cur_idx, return_new=True)[0]
+        current_instance_count = len(current_frame.instances)
+
+        # Get all labeled frames after current frame
+        later_frames = context.labels.frames(video, from_frame_idx=cur_idx + 1)
```

Comment:
If I am at a frame with n>0 instances, I would expect it to jump to a frame where there are 0 instances as well since that is a number of instances change. However, `Labels.labeled_frames` only stores labeled frames with instances>0.